tools/python_api/src_cpp/py_connection.cpp

```diff
@@ -104,8 +104,9 @@ void PyConnection::getAllEdgesForTorchGeometric(py::array_t<int64_t>& npArray,
     auto dstBuffer = buffer + numRels;
     for (int64_t batch = 0; batch < batches; ++batch) {
```

Comment:
If the batch number must be a non-negative number, should we make it an unsigned int?

Reply:
Probably - I've made a small edit to address this, and added a comment for why start and end are int64_t.